repos/fdbt-site/tests/testData/mockData.ts

```diff
@@ -803,6 +803,8 @@ export const userFareStages: UserFareStages = {
     ],
 };
 
+export const fareStagesEditTicket: string[] = ['Acomb Green Lane', 'Mattison Way', 'Nursery Drive'];
```

Comment:
These are fareStage names as strings, so just give it a name to match that. export const fareStageNames: string[]